### DISCUSSION TITLE
Is there a way to use this project with servers that still use the old protocol?

### DISCUSSION THREAD

**Discussion by frederikhors** [enisdenjo/graphql-ws#262] [Q&A] [answered]
After reading https://github.com/enisdenjo/graphql-ws/issues/3 as I am forced to still use [gqlgen](https://github.com/99designs/gqlgen) and therefore the old deprecated protocol (https://github.com/apollographql/subscriptions-transport-ws) I would like to ask you if there is a "legacy" way to be able to use this splendid project instead of the old one while still using an old server like gqlgen (waiting for them to update: https://github.com/99designs/gqlgen/pull/1507).

**Comment by enisdenjo** [comment, marked as answer]
Sadly no, as stated in the [readme disclaimer](https://github.com/enisdenjo/graphql-ws#disclaimer). See also https://github.com/enisdenjo/graphql-ws/issues/154#issuecomment-819368765.
